### ISSUE TITLE
Feature: Support custom device names set in Vysor Server

### ISSUE THREAD

**Issue by vfmach** [koush/vysor.io#1228]
**Desired behavior**
- Set individual device names in the server device modal of Vysor.
- These set names are then properly synced to clients connecting to the server.

**Anti-goals**
- Updated device names on the server do NOT need to sync to the clients in real time, ideally just on re-connect.

**Comment by koush** [comment]
It seems this already exists?

<img width="459" alt="image" src="https://user-images.githubusercontent.com/73924/228731436-a2585d30-e3c3-4400-ab9a-13fb82f18efd.png">


Must be set on the server.

<img width="317" alt="image" src="https://user-images.githubusercontent.com/73924/228731460-a35ac46e-4d5b-4c29-93df-2e04109290f2.png">

**Comment by vfmach** [comment]
@koush I don't believe setting it on the server propagated to the clients when I last tried it -- will try again
